cor_tabular/code/make_table_all.R

```diff
@@ -26,7 +26,7 @@ tlf <-
       the baseline SARS-CoV-2 negative per-protocol cohort",
       table_footer = "This table summarises the random subcohort, which
       was randomly sampled from the per-protocol individuals without a COVID failure
-      event < 7 days post Day 57. The sampling was stratified by the key baseline 
+      event $<$ 7 days post Day 57. The sampling was stratified by the key baseline 
```

Comment:
Is there a reason this is not `$<$`?

Reply:
The change is from < to $<$

Reply:
Ok, it must just be rendering funny on my browser.